sharepoint/sharepoint-ps/sharepoint-online/Set-SPOHomeSite.md

```diff
@@ -32,16 +32,27 @@ Use this cmdlet to set a SharePoint Site as a Home Site. A home site is a commun
 
 ```powershell
 Set-SPOHomeSite -HomeSiteUrl "https://contoso.sharepoint.com/sites/homesite"
-```
 
 This example set the site collection at *<https://contoso.sharepoint.com/sites/homesite>* as SharePoint Online Home Site.
+```
+
+### Example 2
+
+```powershell
+Set-SPOHomeSite -HomeSiteUrl "https://contoso.sharepoint.com/sites/homesite" -VivaConnectionsDefaultStart:$true
+
+Sets the home site to the provided site collection url and keeps the Viva Connections landing experience to the SharePoint home site.
+```
```

Comment:
```suggestion
```
This example set the site collection at *<https://contoso.sharepoint.com/sites/homesite>* as SharePoint Online Home Site.

### Example 2

```powershell
Set-SPOHomeSite -HomeSiteUrl "https://contoso.sharepoint.com/sites/homesite" -VivaConnectionsDefaultStart:$true
```

Sets the home site to the provided site collection URL and keeps the Viva Connections landing experience to the SharePoint home site.
```